README.md

```diff
@@ -2,17 +2,32 @@
 
 This project was generated with [Angular CLI](https://github.com/angular/angular-cli) version 7.3.7.
 
-## Development server
+## Getting Started
 
-Run `ng serve` for a dev server. Navigate to `http://localhost:4200/`. The app will automatically reload if you change any of the source files.
+These instructions will get you a copy of the project up and running on your local machine for development and testing purposes. 
+
+## Prerequisites
+
+* JS
+* TS
+* Node.js
+* Angular
 
-## Code scaffolding
+## Standard Installation
 
-Run `ng generate component component-name` to generate a new component. You can also use `ng generate directive|pipe|service|class|guard|interface|enum|module`.
+1. Make sure all the prerequisites are installed.
+2. Fork dev.to repository, ie. https://github.com/IrinaSendetskaya/coursesApplication/fork
+3. Clone your forked repository, ie. git clone https://github.com/[your-username]/coursesApplication.git
+4. Set up your environment.
+5. Run bin/setup.
+
+## Development server
+
+Run `ng serve` for a dev server. Navigate to `http://localhost:4200/`. The app will automatically reload if you change any of the source files.
 
-## Build
+## Node server
 
-Run `ng build` to build the project. The build artifacts will be stored in the `dist/` directory. Use the `--prod` flag for a production build.
+Go to folder `cd src/db`, run `node server.js` for a server. Navigate to `http://localhost:3000/api/` and `courses` or `users`, or `authors` etc. 
```

Comment:
create a command in package.json  which  do all this at once (run app and server)